import{_ as l,Z as a,$ as c,a0 as e,a1 as n}from"./framework-dee406ed.js";const t={},i=e("h1",{id:"_13-4-自定义包中的错误处理和-panicking",tabindex:"-1"},[e("a",{class:"header-anchor",href:"#_13-4-自定义包中的错误处理和-panicking","aria-hidden":"true"},"#"),n(" 13.4 自定义包中的错误处理和 panicking")],-1),o=e("p",null,"自定义包实现者应该遵守的最佳实践：",-1),_=e("ol",null,[e("li",null,[e("p",null,[e("em",null,"在包内部，总是应该从 panic 中 recover"),n("：不允许显式的超出包范围的 "),e("code",null,"panic()")])]),e("li",null,[e("p",null,[e("em",null,"向包的调用者返回错误值（而不是 panic）。")])])],-1),s=[i,o,_];function r(d,u){return a(),c("div",null,s)}const h=l(t,[["render",r],["__file","13.4.html.vue"]]);export{h as default};
